src/client/sandbox/native-methods.js

```diff
@@ -307,6 +306,12 @@ class NativeMethods {
         if (cssStyleSheetHrefDescriptor)
             this.cssStyleSheetHrefGetter = cssStyleSheetHrefDescriptor.get;
 
+        const nodeBaseURIDescriptor = win.Object.getOwnPropertyDescriptor(win.Node.prototype, 'baseURI');
+
+        // NOTE: only for IE11
```

Comment:
It's difficult to understand by your comment what only IE11 has `baseURI` or only `IE11` does not have this property.
Let's change to `// NOTE: IE11 does not have the 'window.Node.prototype.baseURI' property`

Reply:
`// NOTE: IE11 does not have the 'baseURI' property` 
I think it better)))

And I will replace simular comments into current method.

Reply:
Ok. Rebase this PR